charts/latest/azuredisk-csi-driver/values.yaml

```diff
@@ -25,11 +25,14 @@ image:
     pullPolicy: IfNotPresent
 
 serviceAccount:
-  create: true
+  create: true # When true, a service account will be created for you. Set to false if you want to use your own.
+  controller: csi-azuredisk-controller-sa # Name of Service Account to be created or used
+  node: csi-azuredisk-node-sa # Name of Service Account to be created or used
+  snapshotcontroller: csi-snapshot-controller-sa # Name of Service Account to be created or used
```

Comment:
Since service account creation for the snapshot controller is enabled through `snapshot.snapshotController.serviceAccount`, it would be better to group this value with that setting, e.g. `snapshot.snapshotController.serviceAccountName`

Reply:
good catch, I think we could remove following settings
```
  snapshotController:
    serviceAccount: true
    rbac: true
```

should all depend on
```
serviceAccount:
  create: true

rbac:
  create: true
```

Reply:
OK, I have updated it to reflect the snapshot, controller, and node all depending on serviceAccount.create, and the snapshot rbac depending on rbac.create, and removed these fields from snapshot.snapshotController. I think this is a bit clearer, otherwise I can move the snapshot sa name to snapshot.snasphotController